### DISCUSSION TITLE
[Feature] Update documentation for CLI when using docker

### DISCUSSION THREAD

**Discussion by rmarquet** [immich-app/immich#9664] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

[The CLI page](https://immich.app/docs/features/command-line-interface) mentions using docker but doesn't go into much more detail after that.  I suggest adding something like the following text:

Note, the docker command given doesn't install immich on the system; you have to run that docker command each time.

To run an upload command via docker, you need to do something like the following:

`docker run -it -v "$(pwd)":/import:ro -v your-local-directory-with-assets-to-be-uploaded:/upload:ro -e IMMICH_INSTANCE_URL=http://your-immich-instance/api -e IMMICH_API_KEY=your-api-key ghcr.io/immich-app/immich-cli:latest upload /upload --recursive`

Note the additional -v to specify the directory you want to upload, giving access to that directory to the docker command as /upload (which is then used in the upload command options).

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by psyciknz** [comment]
Having looked for some cli documentation, I was wonder if the CLI instructions needed to include running the CLI within the immich_server or immich_microservices container.

Specifically the api url, which I assumed was localhost:2283/api when in one of these existing containers.  But now I'm writing this, should it have been http://immich_server:2283/api as that is where it exists?
